### PR TITLE
Separate Dockerfile into development and production, and update config to support env variable

### PR DESCRIPTION
**Pull request by jamesy829**

### CODE REVIEW

File: Procfile [hunk 1]
```diff
@@ -1,3 +1,3 @@
-web: bundle exec passenger start -p $PORT -a 0.0.0.0
-worker: bundle exec sidekiq --queue app_perf --verbose
+web: bundle exec rails s -p 5000
```
Comment (randy-girard) [inline Procfile]: Does the port need to be hard coded here?

Comment (jamesy829) [reply Procfile]: It doesn't need to be hard coded, so it can be using `-p $PORT`

File: .env.development.example [hunk 2]
```diff
@@ -1,5 +1,10 @@
-REDIS_URL=redis://localhost:6379/0
-DATABASE_URL=postgres://localhost:5432/app_perf?encoding=utf8&pool=5&timeout=5000
```
Comment (jamesy829) [inline .env.development.example]: Update to have these env as well as redis and postgres gem handles the check of env being available

File: Procfile (hunk 1 above)
Comment (randy-girard) [reply Procfile]: Right, but isn't that what it should be here, so that someone can set `PORT`?

**Pull request merged**